src/gluonts/model/tpp/predictor.py

```diff
@@ -56,10 +56,12 @@ def __call__(
             outputs, valid_length = (
                 x.asnumpy() for x in prediction_net(*inputs)
             )
+            # outputs (num_parallel_samples, batch_size, max_seq_len, 2)
+            # valid_length (num_parallel_samples, batch_size)
```

Comment:
General comment (not strictly concerning the changes in this PR): so the model network is assumed to only produce samples, i.e., there's currently no support for cases were you would want to produce parametric distributions only, is that right?

Reply:
Yes, that is currently the case. As far as I understand, it would only be possible to return the distribution over the inter-arrival time & mark over the next time step (i.e. `p(tau_i | history)` and `p(m_i | history)`). The distribution over `tau_{i+1}` and `m_{i+1}` will depend on the values of `tau_i` and `m_i` that we sampled at the time step `i`.